content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Installing Istio for virtual machine connectivity
+description: Install an Istio mesh such that a virtual machine can connect to it and configure the virtual machine.
+weight: 40
+keywords:
+- kubernetes
+- virtual-machine
+- gateways
+- vms
+---
+
+Follow this guide to install an Istio control plane
+[multicluster deployment](/docs/ops/deployment/deployment-models/#multiple-clusters) and connect your first virtual machine.
```

Comment:
Why does this say mulicluster? If we only do the steps in the following section in only one cluster, it will be a single cluster deployment.

Reply:
The link describes the multiple clusters deployment model. Hence the reference. I can change this if you like.

Reply:
Given this is resolved by your excellent reword, I have clicked resolve.